dynawo/sources/Models/Modelica/Dynawo/Types.mo

```diff
@@ -63,7 +63,11 @@ package Types "Standard types for electrical variables"
   type ApparentPowerPu = PerUnit;
 
   // Percent
-  type Percent = Real (unit = "100") "Percent quantity";
+  type AngularVelocityPu = PerUnit;
```

Comment:
Could you move them just after ApparentPowerPu ? Maybe I was not clear

Reply:
It is not a Percent

Reply:
Already done.